src/identities/mnemonic.ts

```diff
@@ -58,6 +58,9 @@ export class Mnemonic extends Identity implements IdentityInterface {
   readonly masterPrivateKeyNode: BIP32Interface;
   readonly masterBlindingKeyNode: Slip77Interface;
 
+  public masterPublicKey: string;
+  public masterBlindingKey: string;
```

Comment:
why do we want this field `public`?

Reply:
The idea is that we use pass the seed phrase to `Mnemonic` the first time after creation/restore, so we can get the compatible xpubs (signing and blinding) so we can give them to `MasterPublicKey` later on.